Makefile

```diff
@@ -137,6 +137,13 @@ define test_examples
     @echo "# done"
 endef
 
+# run test examples2
+define test_examples2
+    @echo "# Testing examples 2"
+    @./scripts/test_examples2.sh
```

Comment:
i'm sure you will find a better name than test_examples2
add something more descriptive and maybe you can move example related scripts to some folder within scripts folder (i guess there will be more of them later)